Add Descriptions for cards - Card Picker

zsarnett opened the pull request:
## Proposed change


Adds in Descriptions for the other cards in case entities can not be found

zsarnett committed:
Add Descriptions for cards - this is for card pick

zsarnett committed:
oops

frenck commented on src/translations/en.json:
```diff
@@ -1953,7 +1953,8 @@
               "name": "Alarm Panel",
-              "available_states": "Available States"
+              "available_states": "Available States",
+              "description": "The Alarm Panel allows you to Arm and Disarm your Alarm Control Panel Integrations."
```
```suggestion
              "description": "The Alarm Panel allows you to Arm and Disarm your Alarm Control Panel integrations."
```

frenck commented on src/translations/en.json:
```diff
@@ -1966,10 +1967,12 @@
               "show_header_toggle": "Show Header Toggle?",
-              "toggle": "Toggle entities."
+              "toggle": "Toggle entities.",
+              "description": "Entities will be the most common type of card that will also be the most familiar to people using the standard interface. It groups items together very close to how groups used to do."
```
Made it future tense.

I wonder if we should refer to the "old" UI / states UI... Since it is removed by the time this is released. New folks would not have a clue what you are talking about.

```suggestion
              "description": "Entities is the most common type of card that is the most familiar to people using the standard interface. It groups items together, very close to how groups used to do."
```

zsarnett replied:
I agree. We should take the old UI reference out

frenck commented on src/translations/en.json:
```diff
@@ -1966,10 +1967,12 @@
             "button": {
-              "name": "Button"
+              "name": "Button",
+              "description": "The Button card allows you to add buttons to perform tasks."
```
This description starts with "The", while others don't, which is a bit inconsistent.

frenck commented on src/translations/en.json:
```diff
@@ -1982,14 +1985,17 @@
               "name": "Glance",
-              "columns": "Columns"
+              "columns": "Columns",
+              "description": "Glance cards are very compact. Very useful to group together multiple sensors for a quick and easy overview."
```
Stays the same without that additional word.

"very compact" ? Is that clear? Maybe just compact or..

`Glance cards are useful to group multiple sensors in a compact overview.`

```suggestion
              "description": "Glance cards are very compact. Very useful to group multiple sensors for a quick and easy overview."
```

Finally, this is talking about plural cards, while the order sentences are singular (`card` vs `cards`), which is inconsistent.

frenck replied:
Have you considered the alternative I listed?

zsarnett replied:
I believe I changed it to the alternate you listed

zsarnett replied:
`The Glance card is very compact. Very useful to group multiple sensors for a quick and easy overview.`

zsarnett replied:
oh
I see

zsarnett replied:
Changed

frenck commented on src/translations/en.json:
```diff
@@ -2033,26 +2040,33 @@
               "default_zoom": "Default Zoom",
-              "source": "Source"
+              "source": "Source",
+              "description": "A card that allows you to display entities on a map."
```
`A` or `The`?

Furthermore, this description doesn't mention the name of the card in the description, the others do.

frenck commented on src/translations/en.json:
```diff
@@ -2033,26 +2040,33 @@
             "media-control": {
-              "name": "Media Control"
+              "name": "Media Control",
+              "description": "The media control card is used to display Media Player entities on an interface with easy to use controls."
```
The name of the card isn't capitalized in this description, but is in most others. Which is the correct way of naming it?

zsarnett replied:
I believe they should be capital

frenck commented on src/translations/en.json:
```diff
@@ -2033,26 +2040,33 @@
             "picture-entity": {
-              "name": "Picture Entity"
+              "name": "Picture Entity",
+              "description": "Displays the entity in form of an image. Instead of images from URL it can also show the picture of camera entities."
```
```suggestion
              "description": "Displays an entity in the form of an image. Instead of images from URL, it can also show the picture of camera entities."
```

frenck commented on src/translations/en.json:
```diff
@@ -2033,26 +2040,33 @@
             "picture-glance": {
-              "name": "Picture Glance"
+              "name": "Picture Glance",
+              "description": "Show an image card and corresponding entity states as icon. The entities on the right side allow toggle actions, others show the more-info-dialog."
```
Should we write `more-info-dialog`? That seems to bit developers way... `more information dialog` might be more fitting in this context.

```suggestion
              "description": "Show an image card and corresponding entity states as an icon. The entities on the right side allow toggle actions, others show the more-info-dialog."
```

frenck commented:
This is going to help a lot! Nevertheless, left some comments where I think it can be improved.

Consistency (capitalization of card names, mentioning card names or not) is maybe the main point I have.

Another I would like to address (but didn't above), is the use of 'very simple' in quite a view cases. I think it isn't fitting. It could be "not simple" for other people? Maybe a term like "basic" or "elementary" fits better.

zsarnett committed:
Comments from Frenck

zsarnett committed:
few card capitalizations found

zsarnett commented:
@frenck  I agree with you in all these. I have updated most of the descriptions now following a few of your points. They now each say `The [Name of card] card`. I have removed Very Simple cases and references to old UI. Updated a few words choices and long descriptions that didn't make sense.

frenck commented on src/translations/en.json:
```diff
@@ -2033,50 +2040,60 @@
             "thermostat": {
-              "name": "Thermostat"
+              "name": "Thermostat",
+              "description": "The Thermostat card gives control of your Climate entity. Allowing you to change the temperature and mode of the entity"
```
```suggestion
              "description": "The Thermostat card gives control of your climate entity. Allowing you to change the temperature and mode of the entity."
```

zsarnett replied:
You don't think the name of the entity should be capitalized?

zsarnett replied:
Since its a type of entity Climate, Media Player, etc I capitalized them on purpose

frenck replied:
I dunno.. it is not referring to a specific entity or brand, that was kinda my way of thinking in this case.

frenck replied:
Since we don't have guidelines on this, both would be OK. So, whatever fits you. I just came across it while adding full stops at the end of the sentences.

zsarnett replied:
Gotcha I will update them to be lower case. I will go with your method as you are the documentation master. Especially since I want to update the Lovelace Docs to these descriptions when we finish this PR

frenck commented on src/translations/en.json:
```diff
@@ -1982,25 +1985,29 @@
             "iframe": {
               "name": "Webpage",
-              "description": "Embed your favorite webpage right into Home Assistant"
+              "description": "The Webpage card allows you to embed your favorite webpage right into Home Assistant"
```
32325

frenck replied:
```suggestion
              "description": "The Webpage card allows you to embed your favorite webpage right into Home Assistant."
```

zsarnett committed:
Comments

zsarnett committed:
Comments on glance

frenck commented:
✅ Fine by me!
